source/boundary_traction/initial_lithostatic_pressure.cc

```diff
@@ -142,6 +142,9 @@ namespace aspect
       // We do not need the viscosity.
       in0.strain_rate.resize(0);
 
+      // The velocity is zero for the lithostatic case.
```

Comment:
```suggestion
      // The velocity is zero for the lithostatic case. Note that Tensor<1,dim> () initializes a 1D array of length "dim" with zeros.
```

Reply:
Is this better? I didn't want to make the comment too long, since that kind of defeats the purpose of using this shorter version of the constructor that does not include the values of the tensor entries (and I just copied this line from a different place in the code).

Reply:
Yep, this works, thanks!